boilerplate/.eslintrc.js

```diff
@@ -26,6 +26,8 @@ module.exports = {
       },
     ],
     "@typescript-eslint/no-var-requires": 0,
+    "@typescript-eslint/no-require-imports": 0,
```

Comment:
Seems like the new eslint rules from Expo turned this on, so I kept it off for consistency.